components/datetime/src/fieldsets.rs

```diff
@@ -236,6 +238,18 @@ macro_rules! impl_marker_with_options {
                     $(time_precision: yes_to!(options.time_precision, $timeprecision_yes),)?
                 }
             }
+            #[allow(unused)]
+            pub(crate) fn take_from_builder(
```

Comment:
praise: also clever

Reply:
docs nit: since this is macro generated, docs explaining its behavior would be nice